checkov/terraform/checks/resource/ncp/LBTargetGroupUsingHTTPS.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import annotations
+
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class LBTargetGroupUsingHTTPS(BaseResourceCheck):
+
+    def __init__(self):
+        name = "Ensure LB TargetGroup Using HTTPS"
```

Comment:
```suggestion
        name = "Ensure Load Balancer Target Group is not using HTTP"
```
after the previous change, this makes more sense 🙂